lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -311,7 +311,7 @@ private void AddGroupCommunicationOperators()
             try
             {
                 TangFactory.GetTang()
-                    .NewInjector(mapInputPipelineDataConverterConfig)
+                    .NewInjector(mapOutputPipelineDataConverterConfig)
                     .GetInstance<IPipelineDataConverter<TMapOutput>>();
```

Comment:
To make sure that user has set it and its not empty.
On Sep 27, 2015 2:13 AM, "Julia Wang" <notifications@github.com> wrote:

> In lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs
> <https://github.com/apache/incubator-reef/pull/526#discussion_r40501960>:
>
> > @@ -311,7 +311,7 @@ private void AddGroupCommunicationOperators()
> >              try
> >              {
> >                  TangFactory.GetTang()
> > -                    .NewInjector(mapInputPipelineDataConverterConfig)
> > +                    .NewInjector(mapOutputPipelineDataConverterConfig)
> >                      .GetInstance<IPipelineDataConverter<TMapOutput>>();
>
> Do you use the result of the instance of IPipelineDataConverter? What is
> the purpose of the GetInstance() here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/526/files#r40501960>.
>